org.eclipse.m2e.maven.runtime/pom.xml

```diff
@@ -186,14 +187,14 @@
 								org.sonatype.plexus.*;provider=m2e;mandatory:=provider,\
 								org.eclipse.aether.*;provider=m2e;mandatory:=provider;version=${maven-resolver.version},\
 								com.google.inject.*;provider=m2e;mandatory:=provider,\
-								io.takari.*;provider=m2e;mandatory:=provider,\
```

Comment:
I assume that there are some downstream consumers of this package. For them this is probably a breaking change.
In order to allow a migration we should probably We should therefore export the wagon-http packages, maybe even all wagen packages:
`org.apache.maven.wagon*;provider=m2e;mandatory:=provider,\`

Reply:
despite the export, the tests fail to resolve HttpWagon

```
org.eclipse.m2e.core.internal.project.registry.MemoryConsumptionTest.testImportLongBuildChain -- Time elapsed: 0.096 s <<< ERROR!
java.lang.NoClassDefFoundError: org/apache/maven/wagon/providers/http/HttpWagon
```

Reply:
Strange, but I have no immediate idea at the moment.